src/LiveComponent/src/Util/LiveControllerAttributesCreator.php

```diff
@@ -98,6 +98,13 @@ public function attributesForRendering(MountedComponent $mounted, ComponentMetad
             $mountedAttributes = $mountedAttributes->defaults(['data-live-id' => $id]);
         }
 
+        $requestMethod = AsLiveComponent::getMethod($mounted->getComponent());
+
+        if (!isset($mountedAttributes->all()['data-live-request-method-value'])) {
+            $attributesCollection->setRequestMethod($requestMethod);
+            $mountedAttributes = $mountedAttributes->defaults(['data-live-request-method-value' => $requestMethod]);
```

Comment:
Is this line needed? The `$attributesCollection` should, I think, be handling setting this attribute.

Reply:
Line removed